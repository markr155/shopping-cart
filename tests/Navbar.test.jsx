import NavBar from '../src/Components/Navbar';
import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

describe('NavBar renders', () => {
	const { container } = render(<NavBar cart={[]} />, {
		wrapper: BrowserRouter,
	});
	it('renders elements', () => {
		expect(container).toMatchSnapshot();
	});
});
