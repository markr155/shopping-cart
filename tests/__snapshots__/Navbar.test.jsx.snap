// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NavBar renders > renders elements 1`] = `
<div>
  <div>
    <h1>
      Store Name
    </h1>
    <ul>
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/store"
        >
          Store
        </a>
      </li>
      <li>
        <a
          href="/about"
        >
          About
        </a>
      </li>
      <li>
        <a
          href="/cart"
        >
          Cart
        </a>
      </li>
    </ul>
  </div>
</div>
`;
